src/app/visitor/account/tokens.controller.js

```diff
@@ -0,0 +1,138 @@
+angular.module("visitorModule")
+
+.controller('visitorAccountTokensController', [
+    '$scope',
+    '$location',
+    '_',
+    'visitorLoginService',
+    'visitorApiService',
+    'checkoutService',
+    'commonUtilService',
+    function(
+        $scope,
+        $location,
+        _,
+        visitorLoginService,
+        visitorApiService,
+        checkoutService,
+        commonUtilService
+    ) {
+        // General
+        $scope.visitor = visitorLoginService.getVisitor();
+
+        // Tokens List
+        $scope.tokens = [];
+        $scope.remove = remove;
+        $scope.getTokens = getTokens;
+        $scope.popUpOpen = popUpOpen;
+        $scope.showTokenMethod = false;
```

Comment:
Let's use `$scope.tokenableMethods.length` instead of `$scope.showTokenMethod` for checking when to show some blocks in the template